### PR TITLE
cx-993 revised steps options description to outline what are the step…

### PR DESCRIPTION
**Pull request by seewah**
documented steps and options
hid some of the internal details we dont necessarily want to expose

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://185-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-185-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -249,7 +252,22 @@ A number of options are available to allow you to customise the SDK:
   ]
   ```
 
-  In the example above the step `'welcome'` will also pass the values inside of `options` to the properties (`props`) of the React components that make up the step. In order to know which `props` exist for each step, please read the source code for each component. The mapping between steps to components can be found at [StepComponentMap.js][link-to-steps-to-components-map]
+  In the example above, the `title` option of the `welcome` step is being overridden. The list of options that can be overridden is as follows:
+
+  ### welcome ###
+
+  - title
+  - descriptions
```
Comment (rfreitas) [inline README.md]: might be worth stating this is an array

File: README.md [hunk 2]
```diff
@@ -225,11 +225,14 @@ A number of options are available to allow you to customise the SDK:
 
 - **`steps {List} optional`**
 
-  [link-to-steps-to-components-map]:src/components/App/StepComponentMap.js
+  List of the different steps in the flow. Each step is defined by a named string. The available steps are:
```
Comment (stefaniacardenas) [inline README.md]: "Steps in the flow" seems like internal jargon, maybe we could add that each step will display a specific component or add a section with screenshots for each step. What do you guys think?

File: README.md (hunk 1 above)
Comment (seewah) [inline README.md]: ok i will have a look

File: README.md (hunk 2 above)
Comment (seewah) [inline README.md]: i think the concept of steps is kinda important to the JS SDK dev, but i agree it deserves better explanation.  I will try breaking that into subheadings and add more explanation - actually I may even combine that with the "options" bit underneath somehow

**Comment by seewah** [comment]
@rfreitas @stefaniacardenas I have reworked this section to make it more informative. Please review and give feedback. thanks

**Comment by rfreitas** [comment]
Travis automatic deployment: https://185-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-185-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://185-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 3]
```diff
@@ -249,7 +245,35 @@ A number of options are available to allow you to customise the SDK:
   ]
   ```
 
-  In the example above the step `'welcome'` will also pass the values inside of `options` to the properties (`props`) of the React components that make up the step. In order to know which `props` exist for each step, please read the source code for each component. The mapping between steps to components can be found at [StepComponentMap.js][link-to-steps-to-components-map]
+  In the example above, the SDK flow is consisted of three steps: `welcome`, `document` and `face`. Note that the `title` option of the `
+  welcome` step and the `useWebcam` option of the `document` step are being overridden, while the `face` step is not being customised. 
+
+  Below are descriptions of the steps and the custom options that you can specify in the `options` arrays. Unless overridden, the default option values will be used:
+
+  ### welcome ###
```
Comment (stefaniacardenas) [inline README.md]: 👏

File: README.md [hunk 4]
```diff
@@ -249,7 +245,35 @@ A number of options are available to allow you to customise the SDK:
...
+
+  This is the introduction screen of the SDK. Use this to explain to your users that they need to supply identity documents (and face photos) to have their identities verified. The custom options are:
+
+  - title (string)
+  - descriptions (string)
```
Comment (rfreitas) [inline README.md]: it's an array of strings :P maybe `[string]` is enough?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://185-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-185-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
descriptions array issue fixed in last commit

**Pull request merged**